### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by bartimaeusnek**
fixes https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/5412
fixes https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/5200

### CODE REVIEW

File: src/main/java/gregtech/common/items/GT_MetaGenerated_Item_01.java [hunk 1]
```diff
@@ -874,19 +876,31 @@ public boolean onEntityItemUpdate(EntityItem aItemEntity) {
                     Block tBlock = aItemEntity.worldObj.getBlock(tX, tY, tZ);
                     byte tMetaData = (byte) aItemEntity.worldObj.getBlockMetadata(tX, tY, tZ);
                     if ((tBlock == Blocks.cauldron) && (tMetaData > 0)) {
+
+                        if (bw) {
+                            if (aMaterial == Materials.Osmium)
```
Comment (repo-alt) [inline src/main/java/gregtech/common/items/GT_MetaGenerated_Item_01.java:881]: Wouldn't it make sense to just move these materials to GT?

**Comment by repo-alt** [comment]
Using magic constants is good for performance but really bad for maintainability.

File: src/main/java/gregtech/common/items/GT_MetaGenerated_Item_01.java (hunk 1 above)
Comment (bartimaeusnek) [reply src/main/java/gregtech/common/items/GT_MetaGenerated_Item_01.java:881]: they are not materials. they are a special class that has a virtual material attached to them.

Comment (repo-alt) [reply src/main/java/gregtech/common/items/GT_MetaGenerated_Item_01.java:881]: At least maybe add some constants to the GT_BartWorks_Compat? These raw numbers just look weird.

File: src/main/java/gregtech/loaders/postload/GT_MachineRecipeLoader.java [hunk 2]
```diff
@@ -1782,7 +1782,7 @@ public void run2() {
             GT_Values.RA.addAssemblerRecipe(GT_OreDictUnificator.get(OrePrefixes.cableGt01, Materials.VanadiumGallium, 2L), ItemList.Casing_LuV.get(1L), Materials.Polytetrafluoroethylene.getMolten(288), ItemList.Hull_LuV.get(1L), 50, 16);
             GT_Values.RA.addAssemblerRecipe(GT_OreDictUnificator.get(OrePrefixes.cableGt01, Materials.Naquadah, 2L), ItemList.Casing_ZPM.get(1L), Materials.Polybenzimidazole.getMolten(288), ItemList.Hull_ZPM.get(1L), 50, 16);
             GT_Values.RA.addAssemblerRecipe(GT_OreDictUnificator.get(OrePrefixes.wireGt04, Materials.NaquadahAlloy, 2L), ItemList.Casing_UV.get(1L), Materials.Polybenzimidazole.getMolten(288), ItemList.Hull_UV.get(1L), 50, 16);
-            GT_Values.RA.addAssemblerRecipe(GT_OreDictUnificator.get(OrePrefixes.wireGt01, Materials.SuperconductorUHV, 2L), ItemList.Casing_MAX.get(1L), Materials.Polybenzimidazole.getMolten(288), ItemList.Hull_MAX.get(1L), 50, 16);
+            GT_Values.RA.addAssemblerRecipe(GT_OreDictUnificator.get(OrePrefixes.wireGt04, Materials.SuperconductorUV, 2L), ItemList.Casing_MAX.get(1L), Materials.Polybenzimidazole.getMolten(288), ItemList.Hull_MAX.get(1L), 50, 16);
```
[the remark this answers is not in the document]
Comment (Dream-Master) [reply src/main/java/gregtech/loaders/postload/GT_MachineRecipeLoader.java:1785]: fixed

File: src/main/java/gregtech/api/enums/Materials.java [hunk 3]
```diff
@@ -834,7 +833,7 @@
...
+    public static Materials Longasssuperconductornameforuhvwire                             = new Materials( 985, TextureSet.SET_METALLIC       ,   1.0F,      0,  3, 1|2                , 255,255, 255,   0,   "Longasssuperconductornameforuhvwire"                           ,   "Superconductor Base UHV"      ,     0,       0,     10800,  10800,  true,  false,  1,   1,   1, Dyes.dyeWhite      , 1, Arrays.asList(new MaterialStack(Draconium, 6), new MaterialStack(CosmicNeutronium, 7), new MaterialStack(Tritanium, 5), new MaterialStack(Americium, 6)), Arrays.asList(new TC_AspectStack(TC_Aspects.ELECTRUM, 24)));
```
[the remark this answers is not in the document]
Comment (Dream-Master) [reply src/main/java/gregtech/api/enums/Materials.java:836]: fixed

**Comment by 0lafe** [comment]
Draconium in a UHV SC isn't a good choice. It's just an ore you get as early as T2 rocket. If D* isn't used I'd recommend changing it to ichorium dust instead for the magical component

**Comment by Anarack** [comment]
I wouldn't say ichor is much better or a choice. Its about the same tier regardless. Though there isn't really any higher tier magical materials that I can think of.

**Comment by GTNH-Colen** [comment]
Not a fan of using ichor in superconductor recipes, ideally they should stay more tech based though thaumium is used to make the enderium for some lower tier ones but still rather avoid it if possible.

**Comment by draknyte1** [comment]
Could always use some of my ‘magic’ materials that are later game. 🤔

**Comment by GTNH-Colen** [comment]
> Could always use some of my ‘magic’ materials that are later game. 🤔

There are some really interesting materials available under GT++, it's a shame we don't take advantage of them more.

**Comment by Dream-Master** [comment]
Maybe tiering draconium right will be a solution as well.

**Pull request merged**